common/province_terrain/00_province_terrain.txt

```diff
@@ -1032,7 +1032,6 @@
 1575=desert
```

Comment:
So these errors appeared, but maybe I'll fix them.
```
[18:57:46][map.cpp:949]: Province 1497 RESERVED1497 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1498 RESERVED1498 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1502 RESERVED1502 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1504 RESERVED1504 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1578 RESERVED1578 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1626 RESERVED1626 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1864 RESERVED1864 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1866 RESERVED1866 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1876 RESERVED1876 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1877 RESERVED1877 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 1878 RESERVED1878 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2029 RESERVED2029 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2093 RESERVED2093 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2094 RESERVED2094 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2096 RESERVED2096 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2104 RESERVED2104 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2107 RESERVED2107 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2111 RESERVED2111 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2287 RESERVED2287 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2301 RESERVED2301 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 2325 RESERVED2325 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5050 RESERVED5050 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5070 RESERVED5070 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5072 RESERVED5072 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5633 RESERVED5633 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5634 RESERVED5634 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 5637 RESERVED5637 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 6378 RESERVED6378 has no pixels in provinces.bmp
[18:57:46][map.cpp:949]: Province 6379 RESERVED6379 has no pixels in provinces.bmp
```

Reply:
_I'll likely use those ID's to fill in new provinces._ 👍